drivers/modem/Kconfig

```diff
@@ -132,6 +132,12 @@ config MODEM_SIM_NUMBERS
 	  Query the SIM card for the IMSI and ICCID identifiers. This
 	  can be disabled if the application does not use a SIM.
 
+config MODEM_CELL_INFO
+	bool "Enable querying for operator and cell info"
+	depends on MODEM_SHELL
+	default n
```

Comment:
Options are disabled by default so this setting is not needed.

Reply:
Ah, thanks for pointing out. Removed now.